### PR TITLE
Update ballerina init command help description

### PR DESCRIPTION
**Pull request by NatashaWso2**
Fixes https://github.com/ballerina-platform/ballerina-lang/issues/9975

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: cli/ballerina-launcher/src/main/resources/cli-help/ballerina-init.help [hunk 1]
```diff
@@ -12,8 +12,11 @@ DESCRIPTION
      sample Ballerina source file. You can use Ballerina.toml to manage
      the organization setting and version for the project.
      
-     The generated set of artifacts also includes a service skeleton 
-     and a test skeleton that you can use to kick start the project. 
+     If you run the init command in the interactive mode (--i) and
```
Comment (hemikak) [inline cli/ballerina-launcher/src/main/resources/cli-help/ballerina-init.help]: --i  should be -i

Comment (hemikak) [reply cli/ballerina-launcher/src/main/resources/cli-help/ballerina-init.help]: We also have the -interactive flag right ?

File: cli/ballerina-launcher/src/main/resources/cli-help/ballerina-init.help [hunk 2]
```diff
@@ -12,8 +12,11 @@ DESCRIPTION
...
+     create a package with a service or a main function, then the
+     generated set of artifacts includes a service skeleton or a main
+     function skeleton depending on if its a service or a main function
+     and a test skeleton that you can use to kick start the project.
```
Comment (hemikak) [inline cli/ballerina-launcher/src/main/resources/cli-help/ballerina-init.help]: Does this part sound redundant to say ? 

' depending on if its a service or a main function and'

File: cli/ballerina-launcher/src/main/resources/cli-help/ballerina-init.help (hunk 1 above)
Comment (NatashaWso2) [reply cli/ballerina-launcher/src/main/resources/cli-help/ballerina-init.help]: Changed --i to -i and added the --interactive flag with 1cdb231

File: cli/ballerina-launcher/src/main/resources/cli-help/ballerina-init.help (hunk 2 above)
Comment (NatashaWso2) [reply cli/ballerina-launcher/src/main/resources/cli-help/ballerina-init.help]: Yes this sounds redundant so removed that part with 1cdb231

**Pull request merged**